### DISCUSSION TITLE
无法运行程序

### DISCUSSION THREAD

**Discussion by akichannnnn** [Sh1n3zZ/Draw-Lots-Tools#7] [Q&A] [answered]
系统:Armbian
结构:ARM64
CPU:全志H5
在运行的时候报错并且kernel panic

**Comment by Sh1n3zZ** [comment, marked as answer]
使用Windows操作系统重试

**Comment by AkiraKuronagi** [comment]
要解决您在Armbian系统上遇到的内核恐慌问题，请按以下步骤排查。首先，最关键的是想办法拍下或记录屏幕上显示的具体错误信息。然后，优先检查硬件，因为这通常是主因：确保您使用的是功率足够且质量好的电源和数据线，并换用一张可靠的品牌SD卡。同时，检查CPU是否因过热而导致不稳定。如果硬件没有问题，再尝试从软件层面解决，比如运行系统更新命令 sudo apt update && sudo apt full-upgrade，或者通过 armbian-config 工具尝试切换到另一个内核版本。如果以上方法都无法解决，最后的方案就是备份您的重要数据，然后使用最新的稳定版系统镜像进行重装。
